services/explorer-api/src/http-server/routes/controllers/inofficial.ts

```diff
@@ -0,0 +1,99 @@
+import asyncHandler from "express-async-handler";
+import { controllers as db } from "../../../database/index.js";
+import {
+  address,
+  blockHash,
+  blockHeight,
+  heightOrHash,
+  paths,
+  txEffectIndex,
+  txHash,
+} from "../paths_and_validation.js";
+
+const SUB_PATH = "/v1/d1e2083a-660c-4314-a6f2-1d42f4b944f4";
```

Comment:
maybe make a const of this api key so if it chachges we do not have to go trough the code base